app/views/layouts/application.html.erb

```diff
@@ -9,36 +9,39 @@
   </head>
 
   <body>
-    <nav1 class="navbar navbar-expand-lg navbar-light style="background-color: #e3f2fd;">
```

Comment:
Might be a mistake here that I did not catch

Reply:
It looks like all of the `href` values were erased for some reason. You may need to go back to master and copy and paste the original over your update.

Reply:
might try that